app/helpers/application_helper.rb

```diff
@@ -24,7 +24,7 @@ def session_with_trust?
   end
 
   def page_with_trust?
-    return false if current_page?(controller: 'users/sessions', action: 'new')
+    return false if current_page?(controller: '/users/sessions', action: 'new')
```

Comment:
When rendering `layouts/base` from `idv/hybrid_mobile/document_capture_controller`, rails would attempt to resolve `users/session` here as `idv/users/sessions`, which would error out. Prefixing this with a `/` fixed it (which, I did not realize directory-style controller references could do that), but there may be some tweaks to routing _or something_ that could obviate the need for this.